doc/contributing/releases.md

```diff
@@ -285,6 +285,8 @@ You can integrate the PRs into the proposal without running full CI.
 ⚠️ At this point, you can either run `git node release --prepare`:
 
 ```console
+$ # You need to have locally the tag of the latest release of the release line
+$ git fetch http://github.com/nodejs/node.git tag -n vx.b.c
```

Comment:
Usually `git remote update` would do this work, won't it? I think it's simpler than checking what was the last tag.

Reply:
Wouldn't it be better if `git node release` checked/(optionally?) updated tags?

Reply:
I do not want to fetch all the tags from the upstream, I really want to fetch what's really necessary – I'm not always in a place with good unlimited internet. I am not going to run `git remote update`, that's a deal breaker for me.
Updating NCU to check the version number from the changelog instead of a git tag would work.

Reply:
You can also run `git fetch --tags --all`. Although it fetches all tags, it also considers your remote state. So, unless your remote hasn't been updated since 2020, it should be quite efficient.

But, doing it on `git node release` seems good to me as well.

Reply:
> So, unless your remote hasn't been updated since 2020

Believe it or not, that's probably accurate 😅 One day, a long long time ago, I've cloned the repo with `--single-branch`, and since then I mostly only pulled the `main` branch – and until I've tried to prepare a release, that has never caused any problem for me.